### ISSUE TITLE
php artisan route:list -   Unauthorized access - you do not have the necessary permissions to see this page.

### ISSUE THREAD

**Issue by jkirkby91** [tabacitu/dick#50]
As title suggests running

php artisan route:list

artisan console throws an error

  Unauthorized access - you do not have the necessary permissions to see this page.

Can you point me to why this would be, my issue is i've used dick crud to create a resource i can send GET to no POST so want to view the routes laravel can see, which sounds similar to issues/41

**Comment by londoh** [comment]
I just tried a fresh install and see the same error.

Its being thrown by (probably all?) the Dick controllers in the constructor auth check. 
eg in LogController:

`	public function __construct()
	{
		$this->middleware('auth');

		// Check for the right roles to access these pages
		if (!\Entrust::can('view-logs')) {
	        abort(403, 'Log Controller - Unauthorized access - you do not have the necessary permissions to see this page.');
	    }
	}`

regards


l.

**Comment by tabacitu** [comment]
Hi guys! 

It's a known issue, it's caused by the Entrust library. That's why in version 2 (backpack - to be launched this week) we won't be using Entrust any more.

Keep a close eye here: https://github.com/laravel-backpack

**Comment by jkirkby91** [comment]
will there be an upgrade path from dick to backpack?

**Comment by tabacitu** [comment]
Yes, I will write a tutorial on how to migrate. But we're not done improving Backpack yet - the syntax is still changing. Soon :-)

**Comment by jkirkby91** [comment]
can you link me the dev branch of backpack?

**Comment by tabacitu** [comment]
Hi! [Laravel Backpack](https://laravel-backpack.readme.io/docs) is will soon be ready for prime-time. Would love if you could take a look and give some feedback. Thanks, cheers!
